Comments/Controllers/Api/CommentController.cs

```diff
@@ -57,7 +57,7 @@ public IActionResult PutComment([FromRoute] int commentId, [FromBody] ViewModels
 
         // POST: consultations/api/Comment
         [HttpPost("")]
-        public IActionResult PostComment([FromBody] ViewModels.Comment comment)
+        public IActionResult PostComment([FromBody] ViewModels.Comment comment, Session session) //todo: use the session
         {
             if (!ModelState.IsValid)
             {
```

Comment:
they're to support saving comments without being logged in. the work for them wasn't specified in any card, so rather than add more to this branch, I had a new card raised here: https://nicedigital.atlassian.net/browse/EC-896